xml/article_sap_trento.xml

```diff
@@ -928,13 +929,26 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
         <para> Ensure the &t.server; is running. </para>
       </step>
       <step>
-        <para> On the &t.server; host start the update: </para>
-        <screen>&trentoagentprompt;sudo curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install-server.sh | bash
+        <para>Export the <envar>KUBECONFIG</envar> environment variable for the
+          same user that installed K3s:
+        </para>
+        <screen>export KUBECONFIG=/etc/rancher/k3s/k3s.yaml</screen>
+      </step>
+      <step>
+        <para>
+          With the same user that installed K3s, start the update
```

Comment:
This needs to  work with every kubernetes, suggestion:

With the same user that installed Trento server initially, start the update of the Trento server.

Btw. remove host - that's k3s specific.

Reply:
Ok, done.